From: Jiri Kosina <jkosina@suse.cz>
Subject: [PATCH] x86/cpuinfo: Ignore ->initialized member
Patch-mainline: never, kabi
References: bsc#1091543

Shut up.

[js] addded by commit 30bb9811856f667042e746d8033883b1091a46ce.
     struct cpuinfo_x86 is defined only in the setup code, no need to
     worry about larger size.

Signed-off-by: Borislav Petkov <bp@suse.de>

---
 arch/x86/include/asm/processor.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/x86/include/asm/processor.h
+++ b/arch/x86/include/asm/processor.h
@@ -136,7 +136,9 @@ struct cpuinfo_x86 {
 	 * NOTE: only to be used for l1tf mitigation
 	 */
 	u8			x86_cache_bits;
+#ifndef __GENKSYMS__
 	unsigned		initialized : 1;
+#endif
 };
 
 struct cpuid_regs {
